main.go

```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2020-2022 Doc.ai and/or its affiliates.
+// Copyright (c) 2020-2024 Doc.ai and/or its affiliates.
 //
-// Copyright (c) 2023 Cisco Systems, Inc.
+// Copyright (c) 2023-2024 Cisco Systems, Inc.
```

Comment:
The best option to edit this file is append a new copyright holder

For example,
```go
// Copyright (c) 2020-2022 Doc.ai and/or its affiliates.
//
// Copyright (c) 2023 Cisco Systems, Inc
//
// Copyright (c) 2024 ${YOUR COMPANY NAME e.g. Nordix Foundation }. 
```

Reply:
Thanks, Done. (Apparently it changed its name from nordix to openinfra)

Reply:
Nice! Now,  license things look close to reality. Thanks!